lib/flex/FlexController.js

```diff
@@ -38,7 +38,7 @@ class FlexController extends BaseController {
     this.flexService = options.flexService;
     this.projectSetup = new ProjectSetup(this.cliManager.config.paths.project);
     this._metadata = {
-      serviceName: 'Service name is not available.',
+      serviceName: '',
```

Comment:
It seems more appropriate to set the serviceName to null if you do not have it, instead of an empty string.